### PR TITLE
22931 update consume nr api call

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#22931

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by ketaki-deodhar — APPROVED** [review]

File: queue_services/entity-filer/flags.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "flagValues": {
-        "enable-involuntary-dissolution": true
+        "enable-involuntary-dissolution": true,
+        "namex-nro-decommissioned": true
```
Comment (severinbeauvais) [inline queue_services/entity-filer/flags.json:4]: Are these the default FF values? If so, wouldn't it better to disable the feature if LaunchDarkly can't be reached?

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py [hunk 2]
```diff
@@ -33,9 +33,12 @@ def consume_nr(business: Business, filing: Filing, filing_type: str = None):
 
             namex_svc_url = current_app.config.get('NAMEX_API')
             token = AccountService.get_bearer_token()
+            if flags and (flag_on := flags.is_on('namex-nro-decommissioned')):
+                logger.debug('namex-nro-decommissioned flag: %s', flag_on)
```
Comment (severinbeauvais) [inline queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:37]: Do you want to keep this debug statement?

File: queue_services/entity-filer/flags.json (hunk 1 above)
Comment (vysakh-menon-aot) [reply queue_services/entity-filer/flags.json:4]: nope. only in local (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/flags.py#L71)

**Review by severinbeauvais — APPROVED** [review]

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py (hunk 2 above)
Comment (vysakh-menon-aot) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:37]: not really, kept it there to make sure we remove this once we go to production

Comment (severinbeauvais) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:37]: Sounds good.

**Pull request merged**